Source/Redemption/Private/RedemptionPlayer.cpp

```diff
@@ -32,6 +40,11 @@ ARedemptionPlayer::ARedemptionPlayer()
 		{
 			WidgetComponent->SetWidgetClass((WidgetClass.Class));
 		}
+		static ConstructorHelpers::FObjectFinder<UInputAction> SprintActionFinder(TEXT("/Game/InputActions/IA_Sprint"));
+		if (SprintActionFinder.Succeeded())
+		{
+			SprintAction = SprintActionFinder.Object;
+		}
```

Comment:
I think the problem is here, are you sure this path will get you the IA_Sprint?

Reply:
I fixed that and changed it to "/Game/Player/Input/IA_Sprint" but still nothing is getting called.

Reply:
I will download it on my end and see if I can figure it out

Reply:
on the debug I saw it was null, if you check the path you will see that it is "/Game/Player/Input/Actions/IA_Sprint"